### PR TITLE
feat: Add new relayminer config

### PR DESCRIPTION
**Pull request by red-0ne**
* Update `RelayMiner` config values
* Remove flags that should be sourced from config file

### CODE REVIEW

**Review by Olshansk — APPROVED** [review]
LGTM but please wait for @okdas to approve before merging in case I missed anything

**Comment by okdas** [comment]
LGTM! :) We can merge whenever, just need to bump the version of the chart so LocalNet will pull the latest version on our machines.

**Review by okdas — APPROVED** [review]

**Pull request merged**
